releasenotes/notes/url-sanitizer-26c7078e37f395c0.yaml

```diff
@@ -0,0 +1,6 @@
+---
+other:
+  - |
+    Query strings are stripped out from URLs by default when setting URL
+    metadata on a span. This change affects all integrations that store HTTP
+    metadata, like aiohttp, falcon, requests, urllib3.
```

Comment:
dumb question but what the relation is with the existing config option `trace_query_string`?

Reply:
My understanding is that URLs are normally traced by default and should be stripped of the query string because of security concerns mostly. Hence, in order to trace qs's, the `trace_query_string` option needs to be explicitly set to `True` and all the qs's will be traced via a dedicated HTTP meta tag.

Reply:
Yeah I don't think there's consistency at the moment whether the url tag has the query string or not when `trace_query_string` is enabled, the query string tag is always set though.

I don't think this is really intuitive though. If I put `trace_query_string` then I'd expect the url tag to have the query string too.